unpackaged/config/experience_cloud/sharingRules/Contact.sharingRules-meta.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SharingRules xmlns="http://soap.sforce.com/2006/04/metadata">
+    <sharingCriteriaRules>
+        <fullName>Community_Volunteer</fullName>
+        <accessLevel>Edit</accessLevel>
+        <label>Community Volunteer</label>
+        <sharedTo>
+            <group>Community_Volunteers</group>
+        </sharedTo>
+        <criteriaItems>
+            <field>OwnerId</field>
+            <operation>equals</operation>
+            <value>0051D000006uRA5QAM</value>
```

Comment:
We may want to look at this again since Id's will not be the same across orgs. Sorry I should have thought of this when Kat asked me on Friday. We could something like below. Records owned by community Owner role and subordinates share with the Community_Volunteers role.
<img width="1383" alt="Screen Shot 2021-05-17 at 6 18 13 PM" src="https://user-images.githubusercontent.com/34690264/118563310-55c7a480-b73c-11eb-961e-731eb5b30e35.png">